README.md

```diff
@@ -1,6 +1,8 @@
-# traceme
+# hwtracer
 
-A small Rust/C library to trace an arbitrary thread using Intel Processor Trace
-(PT).
+A small Rust/C library to trace a program using CPU tracing technology.
+
+Initially this library will support Intel Processor Trace, but may one day
```

Comment:
"will support" -> "only supports".

", but may one day" -> ". We hope in the future to support alternatives such as Arm's CoreSight."